### ISSUE TITLE
Frontend Dockerization

### ISSUE THREAD

**Issue by krutarth4** [amosproj/amos2023ws04-pipeline-manager#75]
## User story
1. As a developer building application independent of OS
2. So that every team member can start up the frontend

## Acceptance criteria
* Able to deploy frontend through docker file 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been tested
* Feature has been merged into the main branch
* Important parts have been documented in the technology wiki
* All acceptance criteria were met
* Product owner approved features
* Developers agreed to release

**Comment by krutarth4** [comment]
The issue is to provide docker file for the frontend which can install dependencies, build and deploy the frontend.

@eubec @Elementator
